returnn/tf/layers/base.py

```diff
@@ -12,7 +12,7 @@
 import returnn.tf.compat as tf_compat
 import returnn.tf.util.basic as tf_util
 from returnn.tf.util.data import Data, SearchBeam
-from returnn.tf.util.basic import OutputWithActivation, CustomUpdate, reuse_name_scope
+from returnn.tf.util.basic import OutputWithActivation, CustomUpdate, reuse_name_scope, optional_add
```

Comment:
I would prefer to not import too much here, and rather use `tf_util.optional_add` below.